src/Components/Search/SearchSection.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import SearchCard from "./SearchCard";
+
+interface SearchSectionProps {
+  title: string;
+  items: { title: string }[];
+  buttonText: string;
+}
+
+export default function SearchSection({
+  title,
+  items,
+  buttonText,
+}: SearchSectionProps) {
+  return (
+    <section className="mb-8">
```

Comment:
부모 컴포넌트의 div에 gap-8이 생김에 따라 mb-8은 필요가 없어서 제거 부탁드립니다.
추가로 내부 행사와 부스, 그리고 행사 더보기 텍스트와 비율을 똑같이 맟추기 위하여 max-w-screen-lg 추가 부탁드립니다
```suggestion
    <div className="max-w-screen-lg">
```